src/test/java/org/broadinstitute/hellbender/transformers/BQSRReadTransformerUnitTest.java

```diff
@@ -0,0 +1,89 @@
+package org.broadinstitute.hellbender.transformers;
+
+import org.broadinstitute.hellbender.utils.QualityUtils;
+import org.broadinstitute.hellbender.utils.recalibration.RecalDatum;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+public final class BQSRReadTransformerUnitTest extends BaseTest{
+    @Test
+    public void basicHierarchicalBayesianQualityEstimateTest() {
+
+        for( double epsilon = 15.0; epsilon <= 60.0; epsilon += 2.0 ) {
+            double RG_Q = 45.0;
+            RecalDatum RG = new RecalDatum( (long)100000000, (long) (100000000 * 1.0 / (Math.pow(10.0, RG_Q / 10.0))), (byte)RG_Q);
```

Comment:
This specific form of RecalDatum creation happens enough times here it might be worth extracting a private method for it.